lib/stdlib/src/lists.erl

```diff
@@ -933,10 +1031,16 @@ zipwith3(F, [X | Xs], [], [Z | Zs], {pad, {_, Y, _}} = How) ->
 zipwith3(F, [X | Xs], [Y | Ys], [], {pad, {_, _, Z}} = How) ->
     [F(X, Y, Z) | zipwith3(F, Xs, Ys, [], How)].
 
-%% sort(List) -> L
-%%  sorts the list L
+-doc """
+Returns a list containing the sorted elements of `List1`.
+
+_Example:_
 
--doc "Returns a list containing the sorted elements of `List1`.".
+```
+> lists:sort([4,3,9]).
+[3,4,9]
```

Comment:
Should we include a non-integer example?

Reply:
Yes, good idea.